recipes/openldap/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "openldap" %}
+{% set version = "2.4.46" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-{{ version }}.tgz
+  sha256: 41975e79406502c092732c57ef0c2c2eb318d91e8e765f81f5d4ab6c1db727c5
+
+build:
+  number: 0
+  skip: True  # [not unix]
```

Comment:
Ah, that's why the quick appveyor win :P

It would be quite a thing to be able to support windows, but I guess you'll be ADing if you're there...

Reply:
Probably